src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -1798,6 +1798,40 @@
       <summary>Creates an ImplicitStackAllocArrayCreationExpressionSyntax node.</summary>
     </FactoryComment>
   </Node>
+  <Node Name="CollectionCreationExpressionSyntax" Base="ExpressionSyntax">
```

Comment:
'Creation' feels redundant for this to make sense, no?

Reply:
potentially.  we have ObjectCreation and ArrayCreation.  So there was some precedence here.  We'll take through API review on this though to determine.